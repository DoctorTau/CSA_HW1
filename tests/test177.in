588
-47 8941 -5993 4025 -7625 4623 4609 1557 1591 -9338 -2702 -1496 -8895 8910 -59 2002 -1021 337 -4010 -5418 9224 4793 -5608 -623 9723 4698 2940 -5680 -1785 7455 955 1183 -648 1227 -7277 -6208 8383 2314 8738 -4417 -2163 -7060 5551 -8366 7258 7306 1004 -804 9368 7988 3425 -5576 -4493 -3416 5173 948 -5871 -9335 1812 -3198 -6389 -8826 -9665 -6389 4866 -4791 2642 -5845 833 -5365 -6073 4294 -3871 -3056 -7429 7951 -2887 9819 4276 -8077 1226 -4922 -3314 -3434 7807 2105 -9320 3599 9493 -1005 -582 -8353 6472 4029 -272 -3222 -7971 -8873 -5141 -5678 -6713 -2974 -2846 6283 -2739 -5866 -537 -1563 8469 -5333 8990 -2884 9449 7659 -6208 -716 -2221 -1866 6270 -8340 -5267 -4149 -3684 -5973 1532 2903 397 -4109 3541 -2268 -2506 5835 -919 -6688 5733 -3374 -1345 961 -7241 -3009 1696 -7731 -7839 -5940 -1626 5572 6851 6949 988 -4850 -5848 622 240 -7877 4731 4952 -2997 -1108 4458 5865 -642 -1420 -8999 6155 6400 -1491 -394 3594 -2313 176 -5040 1102 -5342 -3255 4601 -9876 -5266 -4797 -1574 -2428 -8323 -226 2904 -7898 -8384 -7729 -6663 -9426 -9932 -3679 -7097 -9432 -9742 -8891 -7233 8609 9125 6369 -2334 2124 5134 8412 -6305 -6241 -1668 -8266 -8349 -5748 8189 2253 -1017 -358 -9969 -9227 -3697 5323 5766 1064 8396 -5040 8313 -2075 4997 296 552 -9641 4428 -5341 1436 -81 8240 4121 5319 -8662 -8711 -4457 -8921 -811 2527 4666 -6636 -4859 7381 -95 3001 -4477 -8969 -8484 7652 6147 7846 -8553 -4460 -3266 -6526 245 8142 -9391 -6831 -9327 6269 722 4668 -8714 7267 8858 7508 -3656 -4788 4955 -2235 -1931 -3595 1982 -7174 -9904 -5053 -6728 -1278 2612 4659 7445 -1317 3714 45 5316 356 -6878 -8576 -7403 5193 -5230 8113 120 -3650 724 -2204 2527 -2512 -1231 -3455 -1114 6827 4618 -2536 -831 4818 -7834 -1005 110 7893 -491 8357 -7305 -729 6428 6462 622 -3546 -7116 8195 -6465 -9457 -6377 1911 4381 2245 -6931 -4694 -3448 -5202 2277 5001 4478 1241 1526 2847 -8159 -2144 5783 -4422 3899 -8776 -696 9105 -5588 -3674 -7952 3078 -2044 -5497 -2032 -9361 -2665 -1380 -4381 9573 6888 7515 6141 -7042 7361 1879 1232 8791 -4767 -2616 -5280 -9674 -6308 6389 8577 7718 -2723 7251 8260 5274 5145 -1230 4532 7343 -932 7882 -8519 8537 5494 -6942 3272 -1993 895 9079 847 7763 3651 5349 7590 -5859 -8034 -8444 -7528 3706 7481 9790 -7833 8134 9425 -5046 9903 -2194 8413 2965 6517 -7758 7517 -9603 9638 -7688 7791 7122 4177 -6986 -8835 -7206 -9131 -514 -5945 6756 2040 -1394 -9715 495 -6882 -5062 7984 -3722 2000 5220 -3850 -4499 -7903 -8516 706 -7555 -173 7647 8438 -4136 -5447 4604 3671 -244 951 6622 -6558 -4600 3408 -9244 -525 -8260 -7393 -6305 4720 3421 6289 -9278 8366 -8240 -8040 6524 401 -9101 478 -3964 3867 576 -8360 6615 2860 3725 -6344 -2425 -6819 548 1402 -6180 212 809 9083 3906 -6359 921 -3818 -212 35 785 -6731 9522 368 3033 9085 -5827 8751 -1569 -5322 9683 300 7951 -6750 25 1133 -4723 7401 -9997 -3637 593 138 561 -2207 5104 -5268 8908 -5548 -3942 1030 -6436 -6925 -8735 -1707 788 9757 2777 -7096 -3760 3388 -6364 -1816 1486 74 -4133 1976 -6459 7645 -7136 -8960 1228 1791 -8740 8800 -5018 -7652 9843 -4157 4509 -6651 -7174 -7379 3571 7672 4362 -535 8414 9481 -6890 -4090 6166 -2910 1894 6059 -7478 -4291 520 -7925 -9120 4637 1485 4965 -677 4478 -9818 5054 4074 5581 -5955 4324 4448 -4930 7508 -3980 