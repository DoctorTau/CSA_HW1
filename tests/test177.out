-9338 -2702 -1496 -4010 -5418 -5608 -5680 -648 -6208 -7060 -8366 -804 -5576 -3416 -3198 -8826 -3056 -4922 -3314 -3434 -9320 -582 -272 -3222 -5678 -2974 -2846 -5866 -2884 -6208 -716 -1866 -8340 -3684 -2268 -2506 -6688 -3374 -5940 -1626 -4850 -5848 -1108 -642 -1420 -394 -5040 -5342 -9876 -5266 -1574 -2428 -226 -7898 -8384 -9426 -9932 -9432 -9742 -2334 -1668 -8266 -5748 -358 -5040 -8662 -6636 -8484 -4460 -3266 -6526 -8714 -3656 -4788 -7174 -9904 -6728 -1278 -6878 -8576 -5230 -3650 -2204 -2512 -1114 -2536 -7834 -3546 -7116 -4694 -3448 -5202 -2144 -4422 -8776 -696 -5588 -3674 -7952 -2044 -2032 -1380 -7042 -2616 -5280 -9674 -6308 -1230 -932 -6942 -8034 -8444 -7528 -5046 -2194 -7758 -7688 -6986 -7206 -514 -1394 -6882 -5062 -3722 -3850 -8516 -4136 -244 -6558 -4600 -9244 -8260 -9278 -8240 -8040 -3964 -8360 -6344 -6180 -3818 -212 -5322 -6750 -5268 -5548 -3942 -6436 -7096 -3760 -6364 -1816 -7136 -8960 -8740 -5018 -7652 -7174 -6890 -4090 -2910 -7478 -9120 -9818 -4930 -3980 